openlibrary/plugins/importapi/code.py

```diff
@@ -8,7 +8,7 @@
 from openlibrary.catalog.marc.marc_xml import MarcXml
 from openlibrary.catalog.marc.parse import read_edition
 from openlibrary.catalog import add_book
-from openlibrary.catalog.get_ia import get_ia, get_marc_ia, get_marc_record_from_ia
+from openlibrary.catalog.get_ia import get_ia, get_marc_ia, get_marc_record_from_ia, get_from_archive
```

Comment:
Why are there so many methods that sound like they do the same thing?
I'll attempt to document and refactor these down.